# Este Ã© um exemplo de arquivo .msg
# Ele define a estrutura de uma mensagem customizada.

# Um campo do tipo string
string detalhe

# Um campo do tipo inteiro de 32 bits
int32 valor